generators/server/templates/src/main/java/package/config/oauth2/OAuth2JwtAccessTokenConverter.java.ejs

```diff
@@ -95,4 +96,36 @@ public class OAuth2JwtAccessTokenConverter extends JwtAccessTokenConverter {
         }
         return false;
     }
+    /**
+     * Extracts jwt claims and sets it to OAuth2Authentication decoded details.
+     * You can get details like:
+     *
+     * <pre>
+     * <code>SecurityContext securityContext = SecurityContextHolder.getContext();
+     *  Authentication authentication = securityContext.getAuthentication();
+     *
+     *  if (authentication != null)
+     *  {
+     *      Object details = authentication.getDetails();
+     *      if(details instanceof OAuth2AuthenticationDetails) {
+     *          Object decodedDetails = ((OAuth2AuthenticationDetails) details).getDecodedDetails();
+     *          if(decodedDetails != null &amp;&amp; decodedDetails instanceof Map)
+     *          {
+     *             String detailFoo = ((Map) decodedDetails).get("foo");
+     *          }
+     *      }
+     *  }
+
+     * </code>
```

Comment:
Please change this example to fit our Java standards - curly brace on same line, space after if should do it. Thanks!